index.bs

```diff
@@ -1075,12 +1112,68 @@ To <dfn export for="SendStream" lt="create|creating">create</dfn> a
 1. Let |stream| be a [=new=] {{SendStream}}, with:
     : [=[[InternalStream]]=]
     :: |internalStream|
-1. [=WritableStream/Set up=] |stream| with TBD.
-1. [=set/Add=] |stream| to |transport|'s [=[[SendStreams]]=].
+    : [=[[Transport]]=]
+    :: |transport|
+1. Let |writeAlgorithm| be an action that [=writes=] |chunk| with |stream|, given |chunk|.
+1. Let |closeAlgorithm| be an action that [=closes=] |stream|.
+1. Let |abortAlgorithm| be an action that [=aborts=] |stream|.
+1. [=WritableStream/Set up=] |stream| with [=WritableStream/set up/writeAlgorithm=] set to
+   |writeAlgorithm|, [=WritableStream/set up/closeAlgorithm=] set to |closeAlgorithm|,
+   [=WritableStream/set up/abortAlgorithm=] set to |abortAlgorithm|.
+1. [=set/Append=] |stream| to |transport|'s [=[[SendStreams]]=].
 1. Return |stream|.
 
 </div>
 
+<div algorithm>
+To <dfn for="SendStream">write</dfn> |chunk| with a {{SendStream}} |stream|, run these steps:
+
+1. Let |transport| be |stream|'s [=[[Transport]]=].
+1. If |chunk| is not a {{Uint8Array}}, return [=a promise rejected with=] a {{TypeError}}.
```

Comment:
In Blink we allow any `BufferSource` type here. Is there a reason to be more restrictive?

Reply:
I don't have a strong reason. I filed #284. Let's discuss there.